Code/wwlib/mutex.h

```diff
@@ -38,8 +40,7 @@
 
 class MutexClass
 {
-	void* handle;
-	unsigned locked;
+    std::mutex mtx;
```

Comment:
The windows "mutex" used here is an inter process object that is best replicate via C++20 semaphores if we are strictly replicating the behaviour of this class. Otherwise any use of it should probably just be replaced with CriticalSectionClass.

Reply:
It creates a syncronisation object that can be shared between processes (not just threads in the same process) which behaviour wise I believe is more inline with what is called a semaphore pretty much everywhere else.